cmd/podman/machine/start.go

```diff
@@ -79,7 +84,35 @@ func start(_ *cobra.Command, args []string) error {
 	if err := vm.Start(vmName, startOpts); err != nil {
 		return err
 	}
+	if err := changeDockerContext(); err != nil {
+		fmt.Printf("Failed to change docker context to default. Error: %v\n", err)
+	} else {
+		fmt.Println("Docker context has been set to default")
+	}
 	fmt.Printf("Machine %q started successfully\n", vmName)
 	newMachineEvent(events.Start, events.Event{Name: vmName})
 	return nil
 }
+
+func changeDockerContext() error {
+	if runtime.GOOS != "darwin" {
+		return nil
+	}
```

Comment:
Do other platform not have the same problem?

Reply:
No, this change is just for MacOS.